products/brms/get-started.adoc

```diff
@@ -158,9 +158,9 @@ http://www.schabell.org/2015/07/workshop-in-can-jboss-brms-full-day-workshop.htm
 Below you will find links to all the materials, articles, blogs, video channels, demo collections and other content that you might want to bookmark as your JBoss BRMS collection.
 
 * link:#{site.base_url}/products/brms/index.html[JBoss BRMS product page]
-* link:#{site.base_url}/products/brms/developer-materials/#!projectCode=brms[Developer materials for JBoss BRMS]
+* link:#{site.base_url}/resources/#!project=brms[Developer materials for JBoss BRMS]
```

Comment:
This should stay in the product pages. So link to the learn page for BRMS. E.g:

      link:#{site.base_url}/products/brms/learn/#!projectCode=brms[Developer materials for JBoss BRMS]